CONTRIBUTING.md

```diff
@@ -61,6 +64,34 @@ Every new file must contain the Eclipse license information and the
 copyright holder(s). Please take a look into existing files and adopt
 the needed changes to your new file(s).
 
+Submit Patches via Github:
```

Comment:
I think that the correct spelling is "GitHub".
(also inconsistent in other places in this document).